src/conductor/input_parser.star

```diff
@@ -63,4 +68,7 @@ def parse(
         "op.conductor.type": "op-conductor",
     }
 
+    # Drop None-valued optional fields to keep output minimal AFTER all defaults are applied
+    conductor_params = _filter.remove_none(conductor_params)
+
```

Comment:
Please remove - all the keys are required even if they are `None` - you are risking invalid property access downstream